src/docs/product/accounts/early-adopter-features/issue-archiving.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "Issue Archiving"
+sidebar_order: 1
+description: "Learn how issue status works and and how to triage issues."
+---
+
+Use the status tags attached to issues on the [**Issues** page](https://sentry.io/issues/) in Sentry to help you triage and prioritize problems with your application that are important to you. Keep in mind that an issue can only have **one status at a time.**
+
+Here's a list of all statuses, how they're assigned to an issue, and their custom search term:
+
+| Status         | Condition                                                                                                                     | Custom Search Term |
+| -------------- | ----------------------------------------------------------------------------------------------------------------------------- | ------------------ |
+| **New**        | An issue that was created in the last 7 days.                                                                                 | is:new             |
+| **Ongoing**    | An issue that was created more than 7 days ago or has manually been marked as reviewed.                                       | is:ongoing         |
+| **Escalating** | An issue that's exceeded its forecasted event volume. For more details, see [Escalating Issues Algorithm](escalating-issues). | is:escalating      |
+| **Regressed**  | A resolved issue that's come up again.                                                                                        | is:regressed       |
+| **Archived**   | An issue that's been marked as archived.                                                                                      | is:archived        |
+| **Resolved**   | An issue that's been marked as fixed.                                                                                         | is:resolved        |
+
+The diagram below shows how the statuses are updated automatically and manually:
+
+![Issue Status Diagram](issue-statuses.png)
+
+One way to limit the issues that you see is by selecting a tab at the top of the **Issues** page. On the “Unresolved" tab, you'll find `New`, `Ongoing`, `Escalating`, and `Regressed` issues. You can also narrow down further by choosing the "For Review", "Regressed", "Escalating", or "Archived" tabs.
+
+## Manually Triaging Issues
+
+While some issue statuses are added and updated automatically, you can manually `Archive` or `Resolve` an issue, which will also change its status.
+
+### Archive
+
+You can manually archive issues that are less pressing or not applicable to you or your team. This will change their status to `Archived` and remove them from the issues list. Sentry will automatically bring an issue back to the top of the list and change its status to `Escalating` if the events in that issue significantly increase over a short period of time. To learn more about how this works, see [Escalating Issues Algorithm](escalating-issues). There's also an option to mark an issue as `Archived` for:
```

Comment:
```suggestion
You can manually archive issues that are less pressing or not applicable to you or your team. This will change their status to `Archived` and remove them from the issues list. Sentry will automatically bring an issue back to the top of the list and change its status to `Escalating` if the events in that issue significantly increase over a short period of time. There's also an option to mark an issue as `Archived` for:
```